.github/workflows/check_nims.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: true
-          token: ${{ secrets.CHECK_OUT_INTERNAL_GH_REPOS_TOKEN }}
+          token: ${{ secrets.GH_REPO_TOKEN }}
```

Comment:
You don't need a token to check out public repos.

Reply:
@bkeryan checks are passing now. I'll abandon #703 and put this one in when you're satisfied.